sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -155,18 +165,42 @@ export class InstitutionController extends BaseController {
     await this.institutionService.syncInstitution(token);
   }
 
+  /**
+   * Controller method to get all institution users with the
+   * given institutionId institution Admin.
+   * @param institutionId institution id
+   * @queryParm page, page number if nothing is passed then
+   * DEFAULT_PAGE_NUMBER is taken
+   * @queryParm pageLimit, limit of the page if nothing is
```

Comment:
Can you give more description for pageLimit, I was able to understand that as page size or records per page.

Reply:
ya, its page size (records per page), will add details to the comment